contentcuration/contentcuration/static/js/edit_channel/views.js

```diff
@@ -530,29 +528,33 @@ var BaseWorkspaceView = BaseView.extend({
 
 var BaseModalView = BaseView.extend({
     callback:null,
-    default_focus_button_selector: null,
+	default_focus_button_selector: null,
+	initialize: function(){
+        _.bindAll(this, "closed_modal", "close")
+	},
     render: function(closeFunction, renderData) {
         this.$el.html(this.template(renderData, {
 			data: this.get_intl_data()
 		}));
         $("body").append(this.el);
```

Comment:
@rtibbles here is where the modals were getting inserted into the DOM.  I noticed that `#dialog` divs were getting added multiple times but were never removed, and that the `#dialog` reference wasn't being used at all for styling.

Regardless, I've now added `id: #dialog` to `BaseModalView`.